src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -428,6 +428,82 @@ private struct WindowManagerSetStylesParams
 
 		#endregion
 
+		#region SetSolidColorBorder
+
+		internal static void SetSolidColorBorder(IntPtr htmlId, string colorHex, string borderWidth)
```

Comment:
One additional change, it'd be best to use a integer instead of a string (it avoids marshalling a string). You can use the same technique as this one: 
https://github.com/unoplatform/uno/blob/c7bc51de97faadd366927c114ec85020a82752e7/src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs#L946